admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -56,7 +56,23 @@ ensure_helper_file_present() {
         fi
     fi
 
-    echo "$DIR/$NAME"
+    readlink -e "$DIR/$NAME"
+}
+
+write_manifests_file() {
+    local TEMPLATE_PATH="$1"
+    local MANIFESTS_FILE="$2"
+
+    if [ -x "$(command -v envsubst)" ]; then
+        envsubst < "$TEMPLATE_PATH" > "$MANIFESTS_FILE"
+    elif [ -x "$(command -v docker)" ]; then
+        TMP_FILE=$(mktemp)
+        export -p > "$TMP_FILE"
+        docker run --rm -v "$TEMPLATE_PATH:$TEMPLATE_PATH" -v "$TMP_FILE:$TMP_FILE" wk88/envsubst "$TEMPLATE_PATH" "$TMP_FILE" > "$MANIFESTS_FILE"
+	rm -f "$TMP_FILE"
```

Comment:
Nit: indentation

Reply:
Also, the temp file won't be removed if `docker run` fails (because of `set -e`)
```bash
local EXIT_STATUS=0 TMP_FILE
TMP_FILE=$(mktemp)
 export -p > "$TMP_FILE" \
  && docker run... \
  || EXIT_STATUS=$?
rm -f "$TMP_FILE"
return $EXIT_STATUS
```
ensures that the temp file gets cleaned up regardless.

Reply:
Solved this with a trap, I think that looks a bit cleaner.